samples/tutorial/sym1/symHello.c

```diff
@@ -0,0 +1,39 @@
+// --------------------------------------------------------------------------
```

Comment:
We don't want any new programs in the tutorial. Please remove it or move it to `targettest/sym1`.

Reply:
Moved to targettest/sym1.